cg/meta/report/templates/balsamic_report.html

```diff
@@ -230,16 +230,16 @@ <h4 class="card-title">Scout</h4>
               <p>Varianter finns uppladdade i Scout: <a href="https://scout.scilifelab.se/{{ customer.id }}/{{ case.name }}">scout.scilifelab.se/{{ customer.id }}/{{ case.name }}</a></p>
               <ul>
                 {% if case.data_analysis.scout_files.snv_vcf != 'N/A' %}
-                  <li><strong>Clinical somatic SNV and INDELs fil</strong> : <em>{{ case.data_analysis.scout_files.snv_vcf.replace(case.id, case.name) }}</em></li>
+                  <li><strong>Kliniskt relevanta förvärvade SNVs och INDELs</strong> : <em>{{ case.data_analysis.scout_files.snv_vcf.replace(case.id, case.name) }}</em></li>
                 {% endif %}
                 {% if case.data_analysis.scout_files.snv_research_vcf != 'N/A' %}
-                  <li><strong>Research somatic SNV and INDELs fil</strong>: <em>{{ case.data_analysis.scout_files.snv_research_vcf.replace(case.id, case.name) }}</em></li>
+                  <li><strong>Förvärvade SNVs och INDELs</strong>: <em>{{ case.data_analysis.scout_files.snv_research_vcf.replace(case.id, case.name) }}</em></li>
                 {% endif %}
                 {% if case.data_analysis.scout_files.sv_vcf != 'N/A' %}
-                  <li><strong>Clinical somatic structural variants fil</strong>: <em>{{ case.data_analysis.scout_files.sv_vcf.replace(case.id, case.name) }}</em></li>
+                  <li><strong>Kliniskt relevanta förvärvade SVs</strong>: <em>{{ case.data_analysis.scout_files.sv_vcf.replace(case.id, case.name) }}</em></li>
                 {% endif %}
                 {% if case.data_analysis.scout_files.sv_research_vcf != 'N/A' %}
-                  <li><strong>Research somatic structural variants fil</strong>: <em>{{ case.data_analysis.scout_files.sv_research_vcf.replace(case.id, case.name) }}</em></li>
+                  <li><strong>Förvärvade SVs</strong>: <em>{{ case.data_analysis.scout_files.sv_research_vcf.replace(case.id, case.name) }}</em></li>
```

Comment:
Could you have a look at these translations in the delivery report @pbiology? Basically it shows which files from Caesar have been uploaded to Scout 

<img width="500" alt="Screenshot 2023-03-29 at 13 24 34" src="https://user-images.githubusercontent.com/26309194/228520824-c8402eb4-df68-4824-8441-6607aa3211e7.png">

Reply:
Should we call the research one "all SNVs and Indels"? To further explain how it differs from the clinical ones?